v2/samples/apimanagement/v1api20220801/v1api20220801_authorizationprovider.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: apimanagement.azure.com/v1api20220801
+kind: AuthorizationProvider
+metadata:
+  name: sampleapimauthorizationprovider
+  namespace: default
+spec:
+  displayName: "sampleauthcode"
+  identityProvider: "aad"
+  oauth2:
+    grantTypes:
+      authorizationCode:
+        clientId: "000000000"
+        clientSecret: "*"
```

Comment:
Can we change it so that it pulls it from a Kubernetes secret instead?

Reply:
its a sample, so I am not sure if that is needed

Reply:
I agree with Tom, the people using this framework are going to be cloud native people and I cant see them using anything but kubernetes secrets